src/vaski-hop.js

```diff
@@ -1,16 +1,55 @@
 'use strict';
 
-var allPlayerImageSelectors, allComputerImgs, scissors, scissors2, paper, paper2, well, well2, continueButton, resetButton, playerSelection, computerSelection, playerName;
+var allPlayerImageSelectors, allComputerImgs, scissors, scissors2, paper, paper2, well, well2, continueButton, resetButton, playerSelection, computerSelection, playerName, winningFigure, selections, gameOutcome;
 
 var typeScissors = 1;
 var typePaper = 2;
 var typeWell = 3;
 
+
+function compareSelections(playerSelection, computerSelection) {
```

Comment:
vėlgi - tu vėl save apriboji liesdamas globalius kintamuosius!
Visą šitą žaidimą tu gali aprašyti neaprašydamas nei vieno globalaus kintamojo, išskyrus window.* ir document.*
Visose funkcijose daryk return ir gražink lokaliai aprašytus kintamuosius (let arba const keywordai).
Realiai net let nėra būtina galima viską aprašyti per const!

Kai pradedi keisti globalius kintamuosius - visas kodas, kuris naudoja (tiesiogiai ar per kitą kodą) šitą funkciją - tampa priklausomas nuo to kiek kartų, kuriuo metu iškviečia šitą funkciją, fakto kad yra globalus kintamasis ir fakto, kad jis pavadintas _winningFigure_.

Reply:
funkcija vadinasi compareSelections - palyginti pasirinkimus. Kas yra palyginimo rezultatas? Kai kažką lygini tu gali padaryti išvadą - A yra daugiau už B, arba A yra lygu B arba A yra mažiau už B. O šitos funkcijos rezultatas yra ...?